gitlab-runner/Cargo.toml

```diff
@@ -37,6 +37,8 @@ rand = "0.8.5"
 
 [dev-dependencies]
 tokio = { version = "1.13.0", features = [ "full", "test-util" ] }
+tracing-subscriber = { version = "0.3.8", features = ["env-filter", "json"] }
```

Comment:
The `EnvFilter` builder API was added in [0.3.10](https://github.com/tokio-rs/tracing/releases/tag/tracing-subscriber-0.3.10).